src/main/resources/templates/server_message.mustache

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Server Message</title>
+    <link rel="stylesheet" href="/css/lib/bootstrap.min.css">
+      <link type="text/css" rel="stylesheet" href="/css/emissary.css"/>
+  </head>
+
+  <body>
+  <header></header>
+  <main class="containter">
```

Comment:
container?

Reply:
looks like a possible c/p from environment.mustache which has the same issue (but I caught it this time)

Reply:
Good catch...fixed